rasa/core/run.py

```diff
@@ -94,6 +94,7 @@ def configure_app(
     syslog_port: Optional[int] = None,
     syslog_protocol: Optional[Text] = None,
     request_timeout: Optional[int] = None,
+    server_listeners: Optional[List[Tuple[Callable, Text]]] = None,
```

Comment:
What are these server listeners for? Asking out of ignorance really 😄

Reply:
they are a more idiomatic way to register hooks to be triggered after the server starts - e.g. for the chat command we want to open the browser after the server is ready.